R/Execution.R

```diff
@@ -56,7 +56,9 @@ execute <- function(analysisSpecifications,
     dir.create(executionScriptFolder)
   }
 
-  executionSettings$databaseId <- createDatabaseMetaData(executionSettings)
+  if ("CdmExecutionSettings" %in% class(executionSettings)) {
```

Comment:
It is recommended to use the `is()` function instead of doing string comparisons on the class attribute (not sure why, but at least is looks cleaner). So this code could read `if (is(executionSettings, "CdmExecutionSettings")) {`